service.kn.switchtimer/resources/language/English/strings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<strings>
```

Comment:
xml based language files are deprecated, please convert them to .po files.
https://github.com/alanwww1/xbmc-xml2po
https://github.com/alanwww1/xbmc-xml2po/downloads